doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -3,6 +3,21 @@
 
 You can use timers to trigger logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
 
+== Selecting the correct timer for your environment
```

Comment:
Really shouldn't use a gerund in this title. What about 
== Types of timers available to use with {PRODUCT} or similar
or
== EBJ and Quartz timers  with {PRODUCT}

Reply:
Changed to "Supported timers for {PRODUCT}"